### PR TITLE
Navbar component

### LINKED ISSUE #17: Should have navigation component

### PR DESCRIPTION
**Pull request by kundanvishen**
This PR will resolve #17 

**Mobile**
screen shot 2018-07-17 at 2 23 25 am
screen shot 2018-07-17 at 2 23 31 am
screen shot 2018-07-17 at 2 23 37 am

**Tablet**
screen shot 2018-07-17 at 2 23 48 am

**Desktop**
screen shot 2018-07-17 at 2 22 33 am

Please ignore color scheme. Navbar has custom color scheme that would be implemented slowly to the rest of the application.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  toggle() {
+    const { isOpen } = this.state;
+    this.setState({
```
Comment (ArfatSalman) [inline src/components/Navbar.js]: Use callback function to set the next state when reading form a previous state.

**Review by ArfatSalman — CHANGES_REQUESTED** [review]

**Comment by kundanvishen** [comment]
@ArfatSalman made the changes. Please approve

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      <div>
+        <Navbar color="dark" dark expand="md">
+          <NavbarBrand href="/">
+            Class Insights
+          </NavbarBrand>
+          <NavbarToggler onClick={this.toggle} />
+          <Collapse isOpen={isOpen} navbar>
+            <Nav className="ml-auto" navbar>
+              <NavItem>
+                <NavLink href="/">
```
Comment (anshurathee) [inline src/components/Navbar.js]: Use the root route from `routes.js`

Comment (kundanvishen) [reply src/components/Navbar.js]: @anshurathee Made the required changes.

**Review by ArfatSalman — APPROVED** [review]

**Pull request merged**